internal/render/pod.go

```diff
@@ -92,6 +93,7 @@ func (p Pod) Header(ns string) model1.Header {
 		model1.HeaderColumn{Name: "READY"},
 		model1.HeaderColumn{Name: "STATUS"},
 		model1.HeaderColumn{Name: "RESTARTS", Align: tview.AlignRight},
+		model1.HeaderColumn{Name: "LAST RESTART", Align: tview.AlignRight, Time: true},
```

Comment:
I think this should be hidden by default.

Reply:
I see that we have a `Hide` bool on the `HeaderColumn` struct, but it seems that is not being used and I didn't find any implementation of it. Will see a way to hide it with only the values!

Reply:
That is correct and will be removed on the next drop. By `hide` I mean making the new column display only while in wide mode. I think it is good additional info but not crucial to be displayed by default.